playbooks/collect-logs.yml

```diff
@@ -136,9 +136,7 @@
     - name: collect logs
       shell: |
         mkdir -p /tmp/{{ inventory_hostname }};
-        for F in $(ls -d1 /var/log/rpm.list /var/log/extra {{ job.archive|join(' ') }}); do
-          cp -rL --parents $F /tmp/{{ inventory_hostname }}
-        done;
+        find {{ job.archive|join(' ') }} -maxdepth 2 -type f -size -1024M -not -path '*/\.*' -exec cp -rL --parents {} /tmp/{{ inventory_hostname }} \;
```

Comment:
Isn't this ignoring `/varlog/rpm.list` and `/var/log/extra` that were mentioned previously?